mmaction/datasets/activitynet_dataset.py

```diff
@@ -156,7 +160,12 @@ def _import_proposals(self, results):
             proposals[video_id] = np.array(this_video_proposals)
         return proposals, num_proposals
 
-    def dump_results(self, results, out, output_format, version='VERSION 1.3'):
+    def dump_results(self,
+                     results,
+                     out,
+                     output_format,
+                     classifier_input=True,
```

Comment:
Need docstring. What does classifier_input mean?

Reply:
` classifier_input` means: whether to generate the classifier (SSN) input file or not.